src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,8 @@
+@function getCardContainerWidth(
+  $product-card-width,
+  $number-of-columns,
+  $column-spase
+) {
+  @return $product-card-width * $number-of-columns
+    + $space-between-columns * ($number-of-columns - 1);
+}
```

Comment:
[STYLES] - If you need to adjust positioning with 1px or fractional numbers like 0.5, you are doing something wrong, change the alignment technique you use